ledger-tool/src/main.rs

```diff
@@ -885,6 +885,12 @@ fn main() {
         .validator(is_parsable::<usize>)
         .takes_value(true)
         .help("How much memory the accounts index can consume. If this is exceeded, some account index entries will be stored on disk. If missing, the entire index is stored in memory.");
+    let accountsdb_skip_shrink = Arg::with_name("accounts_db_skip_shrink")
+        .long("accounts-db-skip-shrink")
```

Comment:
It looks like the majority of the CLI flags are `--no-my-feature` to disable "my-feature". Could this be `--no-accounts-db-shrink` instead? Or is it already `accounts-db-skip-shrink` on the validator CLI?

Reply:
this flag is already in validator